app/routes/_layout+/calendar.tsx

```diff
@@ -0,0 +1,180 @@
+  const authSession = context.getSession();
+  const { userId } = authSession;
+
+  try {
+    // @TODO here we have to handle self-service, and make sure they can only see bookings that belong to them
+    await requirePermission({
+      userId,
+      request,
+      entity: PermissionEntity.booking,
+      action: PermissionAction.read,
+    });
+
+    const header = {
+      title: `Calendar`,
+    };
+
+    return json(data({ header }));
+  } catch (cause) {
+    const reason = makeShelfError(cause);
+    throw json(error(reason), { status: reason.status });
+  }
+};
+export const meta: MetaFunction<typeof loader> = ({ data }) => [
+  { title: appendToMetaTitle(data?.header.title) },
+];
+
+// Calendar Component
+const Calendar = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [title, setTitle] = useState("");
+  const calendarRef = useRef<FullCalendar>(null);
+  const ripple = useRef<HTMLDivElement>(null);
+
+  const handleNavigation = (navigateTo: any) => {
+    const calendarApi = calendarRef.current?.getApi();
+    if (navigateTo == "prev") {
+      calendarApi?.prev();
+    } else if (navigateTo == "next") {
+      calendarApi?.next();
+    } else if (navigateTo == "today") {
+      calendarApi?.gotoDate(new Date());
+    }
+    updateTitle();
+  };
+
+  const updateTitle = () => {
+    const calendarApi = calendarRef.current?.getApi();
+    if (calendarApi) {
+      const currentDate = calendarApi.getDate();
+      const currentMonth = currentDate.toLocaleString("default", {
+        month: "long",
+      });
+      const currentYear = currentDate.getFullYear();
+      setTitle(`${currentMonth} ${currentYear}`);
+    }
+  };
+
+  const toggleSpinner = useCallback(
+    (state: any) => {
+      if (ripple.current) {
+        if (state) {
+          ripple.current.classList.remove("hidden");
+        } else {
+          ripple.current.classList.add("hidden");
+        }
+      }
+    },
+    [ripple]
+  );
+
+  useEffect(() => {
+    updateTitle();
+  }, []);
+
+  return (
+    <>
+      <Header hidePageDescription={true} />
+      <div className="mt-4">
+        <div className="flex items-center justify-between gap-4 rounded-t-md border border-DEFAULT px-4 py-3">
+          <div className="text-left font-sans text-lg font-semibold leading-[20px] text-[#101828]">
+            {title}
+          </div>
+          <div className="flex items-center">
+            <div ref={ripple} className="mr-3 flex justify-center">
+              <Spinner />
+            </div>
+            <ButtonGroup>
+              <Button
+                variant="secondary"
+                className="border-r p-[0.75em] text-[#667085]"
+                onClick={() => handleNavigation("prev")}
+              >
+                <ChevronLeftIcon />
+              </Button>
+              <Button
+                variant="secondary"
+                className="border-r px-3 py-2 text-sm font-semibold text-[#344054]"
```

Comment:
This should be text-gray-700. In figma you can actually see the values.

Reply:
fixed